core/src/test/java/org/apache/druid/common/config/ConfigManagerTest.java

```diff
@@ -49,7 +49,7 @@
 {
   private static final String CONFIG_KEY = "configX";
   private static final String TABLE_NAME = "config_table";
-  private static final TestConfig OLD_CONFIG = new TestConfig("1", "x", 1);
+  private static final byte[] OLD_CONFIG = {1, 2, 3};
```

Comment:
nit: maybe test should serialize test config to bytes to be more realistic?

Reply:
In this test, the byte[] is an input to the function and is just passed to the constructor of the MetadataCASUpdate. The only place that the content of byte[] and the config class matches are in JacksonConfigManagerTest when testing convertByteToConfig. That that test, I serialize test config to bytes to make sure that convertByteToConfig method can serde correctly. In other places, the content of raw byte[] of the config are not important for the test to be actual raw bytes of the config.